### PR TITLE
Update bitpost to 0.9.8.3

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/bitpost.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
   # voluntary.net.s3.amazonaws.com was verified as official when first introduced to the cask
   url "http://voluntary.net.s3.amazonaws.com/Bitpost.#{version}.zip"
-  appcast 'https://voluntary.net/bitpost/'
+  appcast 'https://github.com/VoluntaryLabs/Bitpost/releases.atom'
   name 'Bitpost'
-  homepage 'https://voluntary.net/bitpost/'
+  homepage 'https://github.com/VoluntaryLabs/Bitpost'
```
Comment (vitorgalvao) [inline Casks/bitpost.rb]: Their website is sending me to `github.com/Voluntarynet`, not `github.com/VoluntaryLabs`.

**Comment by vitorgalvao** [comment]
Can’t made heads or tails of this software. There’s no app on the page, it seems to be a web app, now.

**Comment by suschizu** [comment]
you are right - I was searching for an appcast so I came over this page but this project hasn't been updated since 2015 but its still listed on the website: https://voluntarylabs.org/bitpost/index.html - I'm not sure if the voluntary.app is the new version of the bitpost-app … the download link still works so it dosn't seem to be discontinued

**Comment by vitorgalvao** [comment]
Let’s remove it. If someone misses it, we’ll readd it.

**Pull request merged**